.azure/modules/containerAppEnv/main.bicep

```
@@ -13,13 +13,28 @@ param tags object
 @description('The name of the Application Insights workspace')
 param appInsightWorkspaceName string
 
+@description('The Application Insights connection string')
+param appInsightsConnectionString string
+
+@description('The metrics ingestion endpoint of the Azure Monitor workspace')
+param monitorMetricsIngestionEndpoint string
+
+@description('The ID of the user-assigned managed identity')
+param userAssignedIdentityId string
+
 resource appInsightsWorkspace 'Microsoft.OperationalInsights/workspaces@2023-09-01' existing = {
   name: appInsightWorkspaceName
 }
 
-resource containerAppEnv 'Microsoft.App/managedEnvironments@2024-03-01' = {
+resource containerAppEnv 'Microsoft.App/managedEnvironments@2024-02-02-preview' = {
```

Skal denne downgrades? 🤔 
```suggestion
resource containerAppEnv 'Microsoft.App/managedEnvironments@2024-03-01' = {
```

---

yea rev meg ganske mye i håret for å finne ut hvorfor vi ikke kunne spesifisere opentelemetry-config. Det er fordi det fortsatt er i preview. De vil nok få ut final versjon før vi er ute i produksjon tenker jeg, så burde vel være greit? 🤔  Det er tilgjenglig i portalen, bare ikke i Bicep-koden. Fallback om det ikke blir tilfellet er å sende direkte fra koden til azure monitor workspace da 🤔 Ka du tru?

---

🚀 🚀 🚀